### PR TITLE
Add example drivetrain and simulation

### PR DESCRIPTION
**Pull request by jwt388**

### CODE REVIEW

File: src/main/java/frc/robot/DataLogging.java [hunk 1]
```diff
@@ -20,6 +20,8 @@
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
 import edu.wpi.first.wpilibj.smartdashboard.Field2d;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.robot.subsystems.DriveSubsystem;
+
```
Comment (blu28) [inline src/main/java/frc/robot/DataLogging.java:24]: Delete this line.

File: src/main/java/frc/robot/DataLogging.java [hunk 2]
```diff
@@ -137,6 +140,10 @@ public void periodic() {
     if (Constants.LOOP_TIMING_LOG) {
       loopTime.append(Timer.getFPGATimestamp() - startTime);
     }
+
```
Comment (blu28) [inline src/main/java/frc/robot/DataLogging.java:143]: The Contants.LOOP_TIMING_LOG clause needs to be the last thing in the periodic method.

**Review by blu28 — CHANGES_REQUESTED** [review]
Just a couple of small changes. Otherwise it looks good.

File: src/main/java/frc/robot/DataLogging.java (hunk 1 above)
Comment (jwt388) [reply src/main/java/frc/robot/DataLogging.java:24]: Done

File: src/main/java/frc/robot/DataLogging.java (hunk 2 above)
Comment (jwt388) [reply src/main/java/frc/robot/DataLogging.java:143]: Done

**Review by blu28 — APPROVED** [review]
Looks good. You try pushing this PR.

**Comment by blu28** [comment]
Use "Squash and merge" and then delete the branch

**Pull request merged**